### ISSUE TITLE
Check the hierarchy in existing data

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_dev#544]
Consistency checks are all made on the **internal, numeric ids**, the original text-based ids are only consulted after the fact, to identify problems.

```
length(unique(d$processid))
# [1] 123131
length(unique(d$acquisid))
# [1] 138149
length(unique(d$sampleid))
# [1] 102544
length(unique(d$projid))
# [1] 1845
```

### Project level

Each sample, acq and process should be within one and only one project. Inconsistencies for:
- 2 samples (96686 and 123456)
- 38 acqs
- 36 processes

No idea how those came to be.... Oh boy!

### Process

Each process should be nested within one acq, which is within one sample.

8900 processes are associated with more than 1 acq.
1422 are associated with more that 2 acq (i.e. this is not simply the case of d1+d2 in zooscan samples).
Most of those are just called "zooprocess", so it is a case of old versions of zooprocess which did not enforce unicity of ids.
One process called "zooprocess_nan" is associated with 1197 different acq, "apeep" is associated with 960+355 acqs, etc.

399 processes are associated with more than 1 sample
In that case this is mostly related to processes called "MATLAB_PLG_TAKUVIK" and "easyIFCB*something*".
Each of those processes are associated with *thousands* of different samples...

### Acquisitions

Each acquisition should be nested within one sample only.

589 acqs are associated with more than 1 sample.
Most of them are again related to IFCB, with each acquisition being associated with thousands of samples.

### Combinations of sample+acq+process

I don't even want to look at this right now...

**Comment by jiho** [comment]
Start of specification (to be updated)

- [ ] For processes that are linked to several acqusitions, replicate the processes data and link each replicate to one acq and one acq only. Re-recreate the `orig_id` of these processes as `current process orig id`+ "_" + `acq orig_id`
- [ ] For acquisitions that are linked to several samples, replicate the acquisition data and link each replicate to one sample and one sample only. Re-recreate the `orig_id` of these acquisitions as `current acq orig id`+ "_" + `sample orig_id`
- [ ]

**Comment by grololo06** [comment]
Fixed in 2.5.3.
